### PR TITLE
Migrate backend to now v2

### PR DESCRIPTION
**Pull request by pinn3**
Seems to work using `now dev`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 node_modules
 .idea
 .env
-src/key.json
-conversion/public/
\ No newline at end of file
+api/key.json
```
Comment (pinn3) [inline .gitignore]: Tried moving this to the project root, but doesn't look like it'll work without specific configuration for the serverless function as we discussed on slack. Will have to have another go at it

File: api/index.js [hunk 2]
```diff
@@ -1,49 +1,34 @@
-require('dotenv').config()
 const { SESSION_NAME, SESSION_MAX_AGE, SECRET, SITE_URL } = process.env
 
-const { send } = require('micro')
-const morgan = require('micro-morgan')
-const redirect = require('micro-redirect')
 const session = require('micro-cookie-session')({
```
Comment (pinn3) [inline api/index.js]: Was going to replace this library as we'll use the native zeit now request handling, but left it in there as I couldn't find a suitable replacement. Haven't been able to verify that it works as intended though

File: api/index.js [hunk 3]
```diff
@@ -1,49 +1,34 @@
...
 
 const track = require('./lib/track')
 const queryParser = require('./lib/queryParser')
 
-module.exports = morgan('tiny')(async (req, res) => {
-    if (req.url === '/robots.txt') {
-        console.log('Sending robots response')
-        return send(res, 200, ['User-agent: *', 'Disallow: /'].join("\n"))
-    }
-    if (req.url === '/favicon.ico') {
```
Comment (pinn3) [inline api/index.js]: Removed handling of this request as it should really be handled by the `rewrite` rules in the `now.json` configuration, just like the `robots.txt` file. Felt like it was a bit redundant though so left it out

File: now.json [hunk 4]
```diff
@@ -20,6 +26,10 @@
     {
       "source": "/v1/bite.js",
       "destination": "bite.v1.js"
+    },
+    {
+      "source": "/api/robots.txt",
+      "destination": "robots.txt"
```
Comment (pinn3) [inline now.json]: Was also thinking of re-implementing the catch-all route as mentioned [here](https://github.com/barnebys/analytics/pull/9#discussion_r357870926), but wasn't sure of which URL it should point towards

Comment (baboons) [reply now.json]: I will setup an alias for that tomorrow. Just need to come up with decent name/url  :-)

**Comment by pinn3** [comment]
Note to self: remember to remove the section from readme regarding signed requests

File: now.json (hunk 4 above)
Comment (pinn3) [reply now.json]: Let me know when that's ready!

**Pull request merged**